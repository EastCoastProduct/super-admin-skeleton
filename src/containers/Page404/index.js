import React from 'react';
import { IndexLink, Link } from 'react-router';

const Page404 = () =>
  <main>
    <p>This is page 404. Nothing here to see.</p>
    <p>
      Go back to <IndexLink to="/">Home</IndexLink> page
      or <Link to="/login">Login</Link> page.
    </p>
  </main>;

export default Page404;
