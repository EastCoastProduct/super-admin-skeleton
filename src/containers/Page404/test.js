import React from 'react';
import { shallow } from 'enzyme';
import Page404 from './';

describe('Page404 component snapshot', () => {
  it('renders page', () => {
    const wrapper = shallow(<Page404 />);

    expect(wrapper).toMatchSnapshot();
  });
});
