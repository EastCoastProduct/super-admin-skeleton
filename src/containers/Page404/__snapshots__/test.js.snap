exports[`Page404 component snapshot renders page 1`] = `
<main>
  <p>
    This is page 404. Nothing here to see.
  </p>
  <p>
    Go back to 
    <IndexLink
      to="/">
      Home
    </IndexLink>
     page or 
    <Link
      onlyActiveOnIndex={false}
      style={Object {}}
      to="/login">
      Login
    </Link>
     page.
  </p>
</main>
`;
